src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FileReaderService;
+import core.basesyntax.service.FileWriterService;
+import core.basesyntax.service.ReportGeneratorService;
+import core.basesyntax.service.TransactionEvaluatorService;
+import core.basesyntax.service.TransactionParserService;
+import core.basesyntax.service.impl.FileReaderServiceImpl;
+import core.basesyntax.service.impl.FileWriterServiceImpl;
+import core.basesyntax.service.impl.ReportGeneratorServiceImpl;
+import core.basesyntax.service.impl.TransactionEvaluatorServiceImpl;
+import core.basesyntax.service.impl.TransactionParserServiceImpl;
+import core.basesyntax.strategy.CalculationStrategy;
+import core.basesyntax.strategy.impl.CalculationStrategyImpl;
+import java.util.List;
+
+public class Main {
+    private static final String INPUT = "src/main/java/resources/input.csv";
+    private static final String path = "src/main/java/resources/";
```

Comment:
Let's create more informative name  INPUT_FILE_NAME for example.  And constant must be uppercase